src/router/index.js

```diff
@@ -18,6 +19,11 @@ const routes = [
     // which is lazy-loaded when the route is visited.
     component: () =>
       import(/* webpackChunkName: "about" */ "../views/About.vue")
+  },
+  {
+    path: "/error/:id",
+    name: "Error",
+    component: DisplayError
```

Comment:
`DisplayError`, "display error", is a imperative verb expression. Component names should be nouns though.